pkg/controller/forwarding/forwarding_controller.go

```diff
@@ -111,10 +117,13 @@ func (r *ReconcileForwarder) Reconcile(request reconcile.Request) (reconcile.Res
 
 	if instance.Status.IsReady() {
 		instance.Status.Conditions.SetCondition(condReady)
+		r.recorder.Event(instance, "Normal", string(condReady.Type), "All pipelines are valid")
```

Comment:
We don't want to emit an event for every reconcile, only when a condition changes:

```suggestion
		if instance.Status.Conditions.SetCondition(condReady) {
				r.recorder.Event(instance, "Normal", string(condReady.Type), "All pipelines are valid")
		}
```

Reply:
Fixed